kernelcare/manifest.json

```
@@ -0,0 +1,34 @@
+{
+  "display_name": "kernelcare",
```

```suggestion
  "display_name": "KernelCare",
```

We should update the display_name/public_title with the correct way of representing kernelcare's brand/style. There's a few inconsistencies, which will be more difficult to update once this is PR merged.

---

fixed